std/algorithm/searching.d

```diff
@@ -3462,6 +3462,181 @@ unittest
 }
 
 /**
+Computes the index of the first occurrence of `range`'s minimum element.
+
+Params:
+    pred = The ordering predicate to use to determine the minimum element.
+    range = The input range to search.
+
+Returns:
+    The index of the first encounter of the minimum element in `range`. If the
+    range is empty, -1 is returned.
```

Comment:
You are inconsistent here, In the last sentence you don't escape `range`.
FYI function parameters get automatically escaped in ddoc ;-)